### ISSUE TITLE
Feature request - Mod pools for tree generator

### ISSUE THREAD

**Issue by xanthics** [EmmittJ/PoESkillTree#432]
Example: 

chaining Herald of Ice build:
I want ~200% damage in a combination of area, cold, and elemental.  Currently I need to play with various levels of the 3 mods until a tree is generated that I am okay with(to start tweaking) such as 100% elemental, 50% cold, 50% area.  It would be nice to create a "mod group" and then say I want 200% damage and then select those 3 types.

**Comment by brather1ng** [comment]
You can do that already. That's what the "Pseudo Attributes" do:

- On "Pseudo Attributes", add "#% increased Cold Damage" with target 200.
- On "Settings", check the tag "Area".

**Comment by xanthics** [comment]
That "works" and is something that I hadn't gotten working before.

However it seems that all the pseudo attributes get the same settings that define what attributes affect them?  So EG I am playing herald bomber.  I want ~200% cold(ae tagged) and lightning(not ae tagged).  The lightning will always end up being a little lower than anticipated because it also counts ae nodes.  But at least I can play around that by raising lightning damage goals by the amount of area damage selected.

**Comment by brather1ng** [comment]
Yeah, you can't define settings per pseudo attribute. I think the usefulness of that is not worth the time implementing it (would require a complete UI overhaul). Optimizing two skills with different settings at the same time is probably rare.

**Comment by BlazesRus** [comment]
actually you can do that with custom pseudo-attribute things.

**Comment by BlazesRus** [comment]
I have something like that for separate attack elemental area of effect and spell critical chance total on my local data that I use.

**Comment by BlazesRus** [comment]
Edit the Data\PseudoAttributes\ *.xml files to add such things in.

**Comment by BlazesRus** [comment]
I'll upload my pseudo-attributes data with that and some other customized combinations later on GDrive.

**Comment by BlazesRus** [comment]
<PseudoAttribute Name="#% increased Cold Elemental Area of Effect Damage" Group="Damage">
		<Attribute Name="#% increased Elemental Damage" />
		<Attribute Name="#% increased Cold Damage" />
		<Attribute Name="#% increased Area Damage" />
	</PseudoAttribute>
	<PseudoAttribute Name="#% increased Lightning Damage" Group="Damage">
		<Attribute Name="#% increased Elemental Damage" />
		<Attribute Name="#% increased Lightning Damage" />
	</PseudoAttribute>

Heres link to my customized stuff.
https://drive.google.com/file/d/0B8Rugb-pJ6tFN0JoWW02WnhQWEE/view?usp=sharing

**Comment by MLanghof** [comment]
I've used similar customizing before for CoC builds, where you want(ed) to tag "spell" for spell damage scaling, but the critical strike chance you're most interested in is for attacks only.
